gce/Makefile

```diff
@@ -153,6 +182,15 @@ build/prom-conf-url: Makefile local-settings.mk build/prometheus.yml
 	echo "https://storage.googleapis.com/$(GCE_BUCKET)/prometheus.yml" > build/prom-conf-url
 
 gce-create: kubectl calicoctl $(TEMPLATED_OUTPUT)
+	make --no-print-directory deploy-master
```

Comment:
I think you're supposed to use `@(MAKE)` for `make`, then make does some magic to avoid really starting a subprocess